.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: php
+sudo: false
+php:
+  - 7.0
+script:
+  - which pear
+  - find ~ -name pear -print
+  - pear list
+  - pear uninstall xml_util structures_graph archive_tar console_getopt pear
+  - which pear
+  - ./run-installer.sh
+  - find ~ -name pear -print
```

Comment:
@ashnazg yeah, by not utilizing the targets, Travis-CI will execute its defaults. Needless roundtrips and cycles wasted. That's all! :)